src/Makefile.am

```diff
@@ -50,10 +50,8 @@ LIBBITCOIN_WALLET_TOOL=libbitcoin_wallet_tool.a
 endif
 
 LIBBITCOIN_CRYPTO = $(LIBBITCOIN_CRYPTO_BASE)
-if USE_ASM
 LIBBITCOIN_CRYPTO_SSE4 = crypto/libbitcoin_crypto_sse4.la
```

Comment:
It seems there are no reasons to keep `libbitcoin_crypto_sse4`  library anymore. The `crypto/sha256_sse4.cpp` source might be a part of the `libbitcoin_crypto_base`, no?

Reply:
Agreed, but would you be ok with fixing this up in a follow-up? I think it'd make sense to address while dealing with this https://github.com/bitcoin/bitcoin/pull/29404#pullrequestreview-1870581585

Reply:
See #29528.